### ISSUE TITLE
Settings doesn't apply on all tabs (but only on the first one)

### ISSUE THREAD

**Issue by fabifont** [diprajpatra/selenium-stealth#31]
Selenium stealth overrides only the settings of the first tab.

First tab:
![image](https://user-images.githubusercontent.com/38903806/163596356-2358ae05-3f6e-437f-9e99-d54ca59b5003.png)

Second tab:
![image](https://user-images.githubusercontent.com/38903806/163596401-72b66c19-a526-47b8-b6fb-0fde87a58500.png)

Is there a way to override the driver settings globally?

**Comment by adventurebeast** [comment]
I am having this issue as well. In fact. It seems the settings work on the first page load and do not work after that.

**Comment by fabifont** [comment]
@adventurebeast Seems like this project is abandoned

**Comment by adventurebeast** [comment]
@fabifont any suggestions on better projects that are current?

**Comment by fabifont** [comment]
@adventurebeast I don't know actually. But you can ""fix"" that problem by applying stealth to every tab you open.

**Comment by adventurebeast** [comment]
@fabifont thank you! I am new to this. I tried entering the bot.sannysoft.com in the same tab as the driver launched and it failed as well. Oh well. It seems to be working on first launch.

**Comment by adventurebeast** [comment]
Do you have the syntax for applying stealth to every tab that I open?

**Comment by SmartManoj** [comment]
> @fabifont any suggestions on better projects that are current?

https://stackoverflow.com/a/68745328

**Comment by adventurebeast** [comment]
Thanks @SmartManoj. I tried this repo and UC seemed to work similarly. The second tabs on my browser are detected. The first tabs seem to be fine however.

Would still like to find a solution for Selenium Stealth as it performs much faster than UC.

**Comment by BukuBukuChagma** [comment]
@adventurebeast did you find a solution?

**Comment by adventurebeast** [comment]
@BukuBukuChagma I am using Undetected Chromedriver. Seems to work better than selenium stealth for real world applications. It doesnt pass this test either, but it does work better.

**Comment by BukuBukuChagma** [comment]
@adventurebeast ah. I tried that too and it surely works but for my use case I'm also using proxies that require authentication. So, I'm using selenium-wire for that. With undetected-chromedriver I'm not sure how to get the proxies to work. I tried some extension solutions like the one given in top answer in this thread : [stackoverflow](https://stackoverflow.com/questions/55582136/how-to-set-proxy-with-authentication-in-selenium-chromedriver-python) but this works with orignal webdriver but doesn't seem to work with undetected-chromedriver
Ah not really sure what to do lol

**Comment by adventurebeast** [comment]
@BukuBukuChagma Yes I have had the same issues with proxies :/ seems like you are doing what I would do too from the research I have done. I have not tried selenium-wire, but that seems to be the solution.
